q2-structure/proposal-by-xdg.md

```diff
@@ -0,0 +1,278 @@
+# Proposal for Q2: Structure of Governance
+
+## Definitions
+
+This document describes three roles involved in governance:
+
+* "Core Team"
+
+* "Steering Council"
+
+* "Vote Administrator"
+
+A section on each follows.
+
+The term "Perl Rules of Governance" means the final governance document to be
+written and ratified by the Gov-Begin group.
+
+## The Core Team
+
+The Core Team are a group of trusted volunteers involved in the ongoing
+development of the Perl language and interpreter.  They are not required to be
+language developers or committers.
+
+References to specific votes are explained in the "Rules for Voting" section.
+
+### Powers
+
+The Core Team has authority over the rules and composition of Perl governance.
+
+They alone elect the Steering Council and have the power to remove Steering
+Council members.
+
+In concert with the Steering Council, they manage their own membership and
+have the power to modify the Perl Rules of Governance.
+
+The Core Team do not have any authority over parts of the Perl ecosystem
+unrelated to developing and releasing the language itself.  These include, but
+are not limited to:
+
+- The Perl Foundation
+
+- CPAN administration and CPAN authors
+
+- perl.org, metacpan.org, and other community-maintained websites and services
+
+- Perl conferences and events, except those sponsored directly by a
```

Comment:
I think "sponsored" may be a confusing word, since we also talk about donors as conference sponsors. Maybe use something like "organized by the Core Team"?

Reply:
Done.